mesop/examples/web_component/leaflet/leaflet_app.py

```diff
@@ -0,0 +1,64 @@
+"""Proof of concept of a webcomponent wrapper that adds Leaflet map support.
+
+Note: Does not implement full interactions with a map. This is meant as a 
+    proof of concept that others can build upon for additional interactiosn with 
+    a web map.
+"""
+# app.py
+from pydantic import BaseModel
+from typing import Tuple
+
+
+import mesop as me
+import mesop.labs as mel
+from components.leaflet_map_component import leaflet_map_component  # Assuming components directory
```

Comment:
Can you update this to use the full path to the component? Mainly just for consistency since that's how the other examples are written (so they work in the Mesop dev build). Though in the future, we may loosen that requirement.

Reply:
Done!